isaric/parsers/datcov-southafrica.toml

```diff
@@ -49,7 +49,6 @@
 
 [subject]
   # study_id = "datcov-southafrica"
-  country_iso3 = "ZAF"
```

Comment:
The country code should relate (according to Laura) to the study site, rather than the patient. We therefore keep the country code in the visit table, but not the subject - as the subject could feasibly have visited multiple study sites in multiple countries over different visits.